base/multidimensional.jl

```diff
@@ -168,6 +170,7 @@ module IteratorsMD
         error("iteration is deliberately unsupported for CartesianIndex. Use `I` rather than `I...`, or use `Tuple(I)...`")
 
     # Iteration
+    const ORI = OrdinalRange{Int, Int}
```

Comment:
I think this should have a more verbose name, perhaps `OrdinalRangeInt`?

Reply:
Applied.